docs/configurations/day2-config.md

```diff
@@ -1,3 +1,76 @@
 # Day2 configurations
 
-TBD
+## Step 1
+
+First you need to prepare a config map to provide your harbor configurations, apply the config map in the same namespace as harborcluster. In particular, you need to add an annotation to your config map to mark which harborcluster it is acting on.
+
+In addition, in order to protect the password from being displayed directly in the config map, you need to define the password inside the secret, and then specify the name of the secret in the configuration. We currently offer these type of secret configurations fields: `"email_password", "ldap_search_password", "uaa_client_secret", "oidc_client_secret"`.
+
+### Example
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: secret-sample
+  namespace: cluster-sample-ns
+type: Opaque
+data:
+  # the key is same with fields name.
+  email_password: YmFyCg==
+```
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: test-config
+  # namespace same with harborcluster cr namespace.
+  namespace: cluster-sample-ns
+  annotations:
+    # required.
+    # if not define the anno, the config map will not work.
+    # the key is `goharbor.io/configuration`, and the value is your harborcluster cr name.
+    goharbor.io/configuration: harborcluster-sample
+data:
+  # provide your harbor configuration by yaml format.
+  config.yaml: |
+    email_ssl: true
+    email_password: secret-sample # the value is the name of secret which store the email_password.
```

Comment:
Any way to list all the configuration items here? @chlins 

cc @heww

Reply:
Maybe we can link harbor swagger yaml? https://github.com/goharbor/harbor/blob/c701ce09fa2a34b7ea26addb431154f9812de8af/api/v2.0/legacy_swagger.yaml#L516

Reply:
I think it's ok. @chlins